cowin/core/models.py

```diff
@@ -12,7 +15,18 @@ class User(models.Model):
     verified = models.BooleanField(default=False)
     is_deleted = models.BooleanField(default=False)
     message_consent = models.BooleanField(default=False)
+    last_notified_at = models.DateTimeField(null=True)
+
+    def can_notify_now(self):
+        if not self.last_notified_at:
+            return True
+        if (datetime.datetime.now(pytz.UTC) - datetime.timedelta(hours=12)) > self.last_notified_at:
```

Comment:
pytz.UTC is not needed. In settings I have given TIME_ZONE = 'UTC'. Just now() would be enough and then the import would not be required.